docs/en/observability/logs-stream.asciidoc

```diff
@@ -389,4 +394,180 @@ Check the following common issues for possible solutions:
+      }
+    }
+  ]
+}
+----
+
+Now your pipeline will extract these fields:
+
+- The `@timestamp` field – `2023-08-08T13:45:12.123Z`
+- The `log.level` field – `WARN`
+- The `message` field – `192.168.1.101 Disk usage exceeds 90%.`
+
+After creating your pipeline, an index template points your log data to your pipeline. You can continue using the index template you created <<logs-stream-index-template,previously>>
+
+[discrete]
+[[logs-stream-log-level-simulate]]
+=== Test the pipeline with the simulate API
+
+Test that your ingest pipeline works as expected with the {ref}/simulate-pipeline-api.html#ingest-verbose-param[simulate pipeline API]:
+
+[source,console]
+----
+POST _ingest/pipeline/logs-example-default/_simulate
+{
+  "docs": [
+    {
+      "_source": {
+        "message": "2023-08-08T13:45:12.123Z WARN 192.168.1.101 Disk usage exceeds 90%."
+      }
+    }
+  ]
+}
+----
+
+The results should show the `@timestamp` and the `log.level` fields extracted from the `message` field:
+
+[source,JSON]
+----
+{
+  "docs": [
+    {
+      "doc": {
+        "_index": "_index",
+        "_id": "_id",
+        "_version": "-3",
+        "_source": {
+          "message": "192.168.1.101 Disk usage exceeds 90%.",
+          "log": {
+            "level": "WARN"
+          },
+          "@timestamp": "2023-8-08T13:45:12.123Z",
+        },
+        ...
+      }
+    }
+  ]
+}
+----
+
+[discrete]
+[[logs-stream-log-level-query]]
+=== Query logs based on `log.level`
+
+With the `log.level` field extracted, you can query high-severity logs (`WARN` and `ERROR`) that might require your immediate attention, and not be overwhelmed by low-severity logs (`INFO` and `DEBUG`) that aren't immediately important. 
+
+Let's say you have the following logs with varying severities:
+
+[source,log]
+----
+2023-08-08T13:45:12.123Z WARN 192.168.1.101 Disk usage exceeds 90%.
+2023-08-08T13:45:14.003Z ERROR 192.168.1.103 Database connection failed.
+2023-08-08T13:45:15.004Z DEBUG 192.168.1.104 Debugging connection issue.
+2023-08-08T13:45:16.005Z INFO 192.168.1.102 User changed profile picture.
+----
+
+You can add them to your data stream using this command:
+
+[source,console]
+----
+POST logs-example-default/_bulk
+{ "create": {} }
+{ "message": "2023-08-08T13:45:12.123Z WARN 192.168.1.101 Disk usage exceeds 90%." }
+{ "create": {} }
+{ "message": "2023-08-08T13:45:14.003Z ERROR 192.168.1.103 Database connection failed." }
+{ "create": {} }
+{ "message": "2023-08-08T13:45:15.004Z DEBUG 192.168.1.104 Debugging connection issue." }
+{ "create": {} }
+{ "message": "2023-08-08T13:45:16.005Z INFO 192.168.1.102 User changed profile picture." }
+----
+
+The following command queries your documents based on the log level keyword and only shows those with `WARN` or `ERROR` log levels: 
+
+[source,console]
+----
+GET logs-example-default/_search
+{
+  "query": {
+    "terms": {
+      "log.level.keyword": ["WARN", "ERROR"]
```

Comment:
This looks off. It should be `log.level`. I think the reason here is that in the initial template you have not included the ECS base template. @eyalkoren can point you to which templates are needed to make sure things work as expected.